contracts/evmscript/executors/CallsScript.sol

```diff
@@ -25,11 +25,14 @@ contract CallsScript is BaseEVMScriptExecutor {
     * @param _script [ specId (uint32) ] many calls with this structure ->
```

Comment:
Note that upgrading this is also a PITA, for similar reasons to `EVMScriptRegistry`.